### PR TITLE
18674 gcp infrastructure

### PR DESCRIPTION
**Pull request by kialj876**
*Issue:*https://github.com/bcgov/entity/issues/18674

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

File: btr-api/tests/postman/btr-api-DEV.postman_environment.json [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+		},
+		{
+			"key": "realm",
+			"value": "bcregistry",
+			"type": "default",
+			"enabled": true
+		},
+		{
+			"key": "auth_api_url",
+			"value": "https://auth-api-dev.apps.silver.devops.gov.bc.ca",
```
Comment (thorwolpert) [inline btr-api/tests/postman/btr-api-DEV.postman_environment.json:37]: this can go through the gateway

**Review by thorwolpert — COMMENTED** [review]
Looks good to me.
Just a comment on going to auth through the gateway instead of direct

File: btr-api/tests/postman/btr-api-DEV.postman_environment.json (hunk 1 above)
Comment (kialj876) [reply btr-api/tests/postman/btr-api-DEV.postman_environment.json:37]: oh nice okay I'll update

Comment (kialj876) [reply btr-api/tests/postman/btr-api-DEV.postman_environment.json:37]: Oh no wait then I'll need an api key etc -- I think for now I will just leave this

**Pull request merged**